Golem/Jobs.cs

```diff
@@ -73,8 +73,9 @@ public async Task<List<IJob>> List(DateTime since)
         var agreements = await _yagna.Api.GetAgreements(since);
         var invoices = await _yagna.Api.GetInvoices(since);
 
-        foreach(var agreement in agreements.Where(a => a!=null && a.AgreementID!=null))
+        foreach(var shallowAgreement in agreements.Where(a => a!=null && a.AgreementID!=null))
         {
+            var agreement = await _yagna.Api.GetAgreement(shallowAgreement.AgreementID!);
```

Comment:
Can you explain what was the problem?

Reply:
The response from /agreements doesn't fill all the fields in agreement - Demand and Offer are missing. You need to get single one to have that data

Reply:
There is wrong assumption in `GetAgreements` implementation, that this endpoints returns agreements, when in reality it list structures like this:
```
#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
#[serde(rename_all = "camelCase")]
pub struct AgreementListEntry {
    pub id: String,
    pub timestamp: DateTime<Utc>,
    pub approved_date: Option<DateTime<Utc>>,
    pub role: Role,
}
```
I think this should be fixed as well, because it leads to confusion